virt/virtual_machines/advanced_vm_management/virt-configuring-a-watchdog.adoc

```diff
@@ -0,0 +1,15 @@
+[id="virt-configuring-a-watchdog"]
+= Configuring a watchdog
+include::modules/virt-document-attributes.adoc[]
+:context: virt-configuring-a-watchdog
+
+toc::[]
+
+Expose a watchdog device on a virtual machine instance (VMI), alongside liveness and readiness probes, to ensure the operating system is running properly.
+
+== Prerequisites
+
+* Kernel support for `i6300esb` inside the guest
```

Comment:
The way this is phrased, it sounds like the kernel has to support the Intel i6300esb chipset. But what it seems to indicate is that the kernel must support the default watchdog device, which emulates the PCI i6300esb. 

See https://www.ovirt.org/develop/sla/watchdog-device.html 

Suggestion: Kernel support for an `i6300esb` watchdog device.

You don't need "inside the guest" because that is the only relevant place for this kernel support to be.